pkg/config/generate.go

```diff
@@ -341,6 +341,36 @@ func Generate(cs *acsapi.ContainerService, c *Config) (err error) {
 		}
 	}
 
+	// random string based configurables
+	strSecrets := []struct {
+		secret *string
+		l      int
+	}{
+		{
+			secret: &c.RegistryConsoleOAuthSecret,
+			l:      68,
+		},
+		{
+			secret: &c.RouterStatsPassword,
+			l:      10,
+		},
+		{
+			secret: &c.RegistryStorageAccount,
```

Comment:
this is not a secret but uses the same pattern. tempted to rename all this to `strConfigurables` or smth like that.

Reply:
they're all subtly different - don't try to combine them.  storage accts must be [a-z0-9]{24} according to https://docs.microsoft.com/en-us/azure/architecture/best-practices/naming-conventions#storage , for now stick with upstream for the registry console secret (user[a-zA-Z0-9]{64}), etc.